clamdscan/client.c

```diff
@@ -175,7 +175,7 @@ int16_t ping_clamd(const struct optstruct *opts)
     char *errchk                = NULL;
     uint64_t i                  = 0;
     const struct optstruct *opt = NULL;
-    int64_t sockd;
+    int64_t sockd               = 1;
```

Comment:
Could this cause an issue with line 266?

Reply:
omg was supposed to be -1.